package.json

```diff
@@ -43,6 +45,13 @@
     "appdmg": "0.4.5"
   },
   "devDependencies": {
+    "babel-plugin-transform-class-properties": "^6.19.0",
```

Comment:
I saw that, but it's not quite working properly yet. I'll leave this in for now and revisit later.